competition-simulator.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: page
+title: Competition Simulator
+---
+
+Competition Simulator
+============
+
+![]({{ site.baseurl }}/resources/competition-simulator/arena.png)
+
+For SR2020, a virtual competition will be run using a brand new simulator.
+
+<div class="info">
+  This is different to the <a href="/docs/programming/simulator/">regular simulator</a>.
+</div>
+
+## Installation
+
+### Prerequisites
+
+This simulation is based in [Webots](https://cyberbotics.com/#download), which will need to be downloaded and installed.
+
+You will also need Python installed. The simulator supports >=3.5. Additional external libraries are not supported.
+
+### Installing the simulation
+
+1. [Download the simulation](https://github.com/srobo/competition-simulator/archive/0.1.0.zip), and unzip it somewhere on your computer.
+2. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+
+You may receive a warning about your computer's GPU not being good enough, which can be ignored.
+
+#### Changing your version of Python
+
+If webots is picking up the incorrect version of Python, you'll need to change it. This can be done using `Tools > Preferences > General > Python command`. You'll need to ensure a matching version of Python is installed.
+
+## Overview
+
+Within the IDE, there are a few different panels:
+
+- In the centre of your screen is the 3D simulated view of the arena
+- On the left is a tree hierarchy of all elements in this "world"
+- On the right is your text editor for modifying the `sr_controller` code for example
+- At the bottom is your console
+- At the top are your general controls which include the time controls. Press the centre play button to run the simulation at normal speed.
+
+### Useful links
+
+- [Camera Controls](https://www.cyberbotics.com/doc/guide/the-3d-window#navigation-in-the-scene)
+- [Graphics settings](https://www.cyberbotics.com/doc/guide/preferences#opengl) (Useful if Webots is running slowly)
+
+## Developing your code
+
+On first run, the robot will execute an example program for convenience. On first run, this will be copied to the directory `competition-simulator` is stored in:
+
+```
+.
+├── competition-simulator
+│   ├── ...
+│   └─ worlds
+│       └── Arena.wbt
+└── robot.py
+```
+
+Your code should be developed in `robot.py`.
+
+
+<div class="warning">
+  Only your controller code will be present in the competition environment.
+</div>
+
+<div class="warning">
+  You will be responsible for backing up and versioning your code, and collaborating with your fellow team members. The <a href="https://studentrobotics.org/ide/">IDE</a> is unsuitable for this simulator.
+</div>
+
+### Updates
+
+Occasionally, we may release an update to the simulation. To update, you will need to delete the `competition-simulator-master` directory, and re-download it using the above link.
+
+## Robot
+
+![]({{ site.baseurl }}/resources/competition-simulator/robot-front.png)
```

Comment:
Should we say something about the robot?

Things I think we should mention here:
- that this is a pre-built shell, which supports our API
- that building a robot model is complicated and we expect teams to use the pre-built shell
- maybe some comments about what it can do?